### ISSUE TITLE
add option to counting

### ISSUE THREAD

**Issue by ercius** [OpenChemistry/stempy#212]
Can we add an option to electron_count which will bypass the threshold calculation?

It would be great to have keywords for `background_threshold` and `xray_threshold`. These are meant to be the values which are currently determined by the `sigma` inputs with Gaussian fitting.

If the new parameters are input then the Gaussian fitting is skipped and the values are used directly. It would essentially skip the line `res = _image.calculate_thresholds(*args)` and use the thresholds directly from the user.

**Comment by cjh1** [comment]
@ercius Should we implement this, shouldn't bee too hard todo.

**Comment by ercius** [comment]
Yes, please. I think it would be great to have more fine grained control.
